### ISSUE TITLE
Pie chart

### ISSUE THREAD

**Issue by ontrou** [phibr0/obsidian-charts#4]
Is there any chance that you'll introduce a pie chart type?

**Comment by phibr0** [comment]
I could add a Pie Chart to this Plugin, but there is already a Pie Chart in Mermaid which comes with Obsidian by default.
https://mermaid-js.github.io/mermaid/#/pie
```markdown
    ```mermaid
    pie title Pets adopted by volunteers
        "Dogs" : 386
        "Cats" : 85
        "Rats" : 15
    ```
```

**Comment by ontrou** [comment]
I do use Mermaid now. But I am looking at your roadmap to be able to create a chart from a table, which would make charts from your plugin much more useful.

The other thing I dislike about Mermaid pie charts is that the way it is labelled. It is sometimes difficult to differentiate the default colours, which you have to do to identify the labels. I've noticed Chartist directly labels its pie charts.

**Comment by phibr0** [comment]
I will take a look into adding Pie Chart support now. To generate the Chart from a Table i would somehow need to parse a table which I dont know how yet.

**Comment by phibr0** [comment]
I added the Pie Chart with the newest Update. Take a look at the Readme for further instructions please.

**Comment by ontrou** [comment]
Thank you so much @phibr0 It looks great.

BTW, in the Readme, you've written: "The Line Chart works a bit differently, it only accepts..." Of course you meant "The Pie Chart..."

**Comment by phibr0** [comment]
Thanks! Fixed the Readme in 711cbc1dc1918b07dccbdba51b52ad86fb53676a
